doc/api/zlib.md

```diff
@@ -712,6 +712,44 @@ The `zlib.bytesWritten` property specifies the number of bytes written to
 the engine, before the bytes are processed (compressed or decompressed,
 as appropriate for the derived class).
 
+### `zlib.crc32(data[, value])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `data` {string|Buffer|TypedArray|DataView} When `data` is a string,
+  it will be encoded as UTF-8 before being used for computation.
```

Comment:
I noticed that the popular npm package https://www.npmjs.com/package/crc-32 will use UTF16 by default (well, since it needs to operate directly on the string for speed, and the only way to iterate over it quickly on the web is to do `charCodeAt`).  I am not sure if users would be surprised by our difference here, but then in most of our APIs like the crypto hashing functions, we generally default to UTF8 as the input encoding. Another solution is to document it better by explicitly use a `Buffer.from('hello', 'utf16')` example below..

Reply:
Added a note about using it with third-party libraries.